Apps/WebClient/src/ClientApp/src/components/report/covid19.vue

```diff
@@ -0,0 +1,265 @@
+            .toPdf()
+            .get("pdf")
+            .then((pdf: PDFDefinition) => {
+                // Add footer with page numbers
+                var totalPages = pdf.internal.getNumberOfPages();
+                for (let i = 1; i <= totalPages; i++) {
+                    pdf.setPage(i);
+                    pdf.setFontSize(10);
+                    pdf.setTextColor(150);
+                    pdf.text(
+                        `Page ${i} of ${totalPages}`,
+                        pdf.internal.pageSize.getWidth() / 2 - 55,
+                        pdf.internal.pageSize.getHeight() - 10
+                    );
+                }
+            })
+            .save()
+            .output("bloburl")
+            .then((pdfBlobUrl: RequestInfo) => {
+                fetch(pdfBlobUrl).then((res) => {
+                    res.blob().then(() => {
+                        this.isLoading = false;
+                        this.isPreview = true;
+                    });
+                });
+            });
+    }
+}
+</script>
+
+<template>
+    <div>
+        <LoadingComponent :is-loading="isLoading"></LoadingComponent>
+        <div ref="report">
+            <section class="pdf-item">
+                <div v-show="!isPreview">
+                    <div id="pageTitle">
+                        <h3 id="subject">Health Gateway COVID-19 History</h3>
+                    </div>
+                    <div id="disclaimer" class="mr-1">
+                        This record was generated by the BC Provincial Health
+                        Gateway application. For any questions, please contact
+                        HealthGateway@gov.bc.ca
+                    </div>
+                    <hr class="mb-0" />
+                    <b-row class="pt-2">
+                        <b-col>
+                            <label>Name:</label> <span>{{ name }}</span>
+                        </b-col>
+                    </b-row>
+                    <b-row class="pt-2">
+                        <b-col>
+                            <label>Date Reported:</label>
+                            <span>{{ currentDate }}</span>
+                        </b-col>
+                    </b-row>
+                </div>
+
+                <b-row v-if="isEmpty" class="mt-2">
+                    <b-col>No records found.</b-col>
+                </b-row>
+                <b-row v-else class="py-3 mt-4 header">
+                    <b-col class="col">Date</b-col>
+                    <b-col class="col">Test Type</b-col>
+                    <b-col class="col">Test Location</b-col>
+                    <b-col class="col">Result</b-col>
+                </b-row>
+                <b-row v-for="item in records" :key="item.id" class="item py-1">
+                    <b-col
+                        data-testid="covid19ItemDate"
+                        class="col my-auto text-nowrap"
+                    >
+                        {{ formatDate(item.collectedDateTime) }}
```

Comment:
it should be item.labResults[0].collectedDateTime

Reply:
fixed. thanks @tiagograf-AOT